__mocks__/cognito-pagination-fake.js

```diff
@@ -0,0 +1,33 @@
+export default class CognitoPaginationFake {
+  constructor() {
+    this.calledAlready = false
```

Comment:
This makes sure we can split a result set into paginated chunks without endless recursion.

Reply:
ditto on putting this comment in the code